Python/lgmpy/Lgm_Vector.py

```diff
@@ -11,20 +11,25 @@
 silly for much of this).
 
 """
-import itertools
+from __future__ import division
+import sys
 import copy
 from ctypes import pointer, c_double
 
 import numpy as np
 
-from Lgm_Wrap import Lgm_Vector, Lgm_VecSub, Lgm_ScaleVector, Lgm_NormalizeVector, \
+from .Lgm_Wrap import Lgm_Vector, Lgm_VecSub, Lgm_ScaleVector, Lgm_NormalizeVector, \
     Lgm_CrossProduct, Lgm_Magnitude, Lgm_ForceMagnitude, Lgm_DotProduct, \
     Lgm_VecDiffMag, Lgm_VecAdd, Lgm_SphToCartCoords, Lgm_CartToSphCoords, Lgm_Slerp, \
     Lgm_InitSlerp
 
 __author__ = 'Brian Larsen (Python), Mike Henderson (C) - LANL'
 
 class Lgm_Vector(Lgm_Vector):
+    if sys.version_info.major<3:
+        _typetup = (int, float, long)
+    else:
+        _typetup = (int, float)
```

Comment:
This seemed like the easy way to do it when I was fixing, as the code already checked that type was one of the members of a tuple. This change just set the tuple at the top of the class and drops `long` under Python 3.
if you have a cleaner suggestion then I'm happy to implement - this was the minimal fix without refactoring the class.

Reply:
My thought was:
```python
try:
    _typetup = (int, float, long)
except NameError: #Python 3
    _typetup = (int, float)
```
It obscures things if there's a _different_ `NameError`, of course....

Reply:
Leaving this _as-is_ for now. The python wrappers need a major overhaul and I doubt this will survive that. It currently works as intended, so marking as resolved.